app/src/main/java/com/example/memeitupapp/data/service/api/MemeItUpApi.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.memeitupapp.data.service.api
+
+import com.example.memeitupapp.data.service.response.MemeBaseResponse
+import com.example.memeitupapp.data.service.response.MemeResponse
+import retrofit2.Call
+import retrofit2.http.GET
+
+interface MemeItUpApi {
```

Comment:
Pls Marcos, rename all the classes/interfaces with that name...it is not representative name.

Reply:
renamed, thanks